canvas_cli/apps/auth/utils.py

```diff
@@ -65,14 +83,16 @@ def get_default_host(host: str | None = None) -> str:
         (host for host in hosts if config.getboolean(host, "is_default", fallback=False) is True),
         hosts[0],
     )
-    if first_default_host == 'localhost':
+    if first_default_host == "localhost":
```

Comment:
In ruby `'` and `"` are slightly different. You only use `"` if you're going to do string interpolation. This is very engrained into me, so I'm always using `'` by default.

Reply:
Are we using black and isort on this project?

I always type `'` to avoid an extra keystroke (shift), and then let the autoformatter deal with it.

Reply:
It appears we are and I just haven't installed the pre-commit hooks. My bad.

Reply:
I appreciate your focus on developer efficiency, btw.

Reply:
Sometimes laziness and efficiency are the same thing